### DISCUSSION TITLE
Pouvoir se libérer soit même de prison

### DISCUSSION THREAD

**Discussion by Bombafox** [Crownicles/Crownicles#1164] [Nouvelles idées]
Au lieu de demander à quelqu'un de payer pour nous libérer 

Pouvoir payer un avocat pour nous sortir nous même
Et genre on paye 1500 💰 et on a 1/2 chance que ça passe
{(50/50) parce que oui dans la vie ça marche pas tout le temps}

**Comment by Pietagorh** [comment]
Incroyable je trouve
J'serais plus pour le mettre à 1750/2000 ou un truc du genre, pour le rendre moins bien que l'unlock

**Reply by romain22222** [reply]
Perso je pense si on le fziy on laisse une seule chance à l emprisonné, et on laisse 3k

**Reply by BastLast** [reply]
Perso je suis pas chaud
